reactular.tsx

```diff
@@ -44,20 +48,22 @@ export const reactular = <Props extends object>(
 
           const Wrapper =
             typeof wrapper === 'string'
-              ? this.$injector.get<React.ComponentType>(wrapper)
+              ? this.$injector.get<
+                  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                  React.ComponentType<React.PropsWithChildren<any>>
```

Comment:
Use the same type from above here

```suggestion
                  React.ComponentType<React.PropsWithChildren<{}>>
```